cli/cmd/plugin/ui/web/tanzu-ui/src/state-management/stores/Store.umc.tsx

```diff
@@ -5,24 +5,26 @@ import React, { createContext, ReactNode, useReducer } from 'react';
 import { STORE_SECTION_FORM } from '../reducers/Form.reducer';
 import { StoreDispatch } from '../../shared/types/types';
 import umcReducer from '../reducers/Wizard.reducer';
+import { UNMANAGED_CLUSTER_FIELDS } from '../../views/unmanaged-cluster/unmanaged-cluster-wizard-page/UnmanagedCluster.constants';
 
 const initialState = {
     [STORE_SECTION_FORM]: {
         // Cluster settings basic
-        CLUSTER_NAME: '',
+        [UNMANAGED_CLUSTER_FIELDS.CLUSTER_NAME]: 'unmanged-cluster',
 
         // Cluster Settings advanced
-        CONTROL_PLANE_NODES_COUNT: 1,
-        WORKER_NODES_COUNT: 0,
-        CLUSTER_PROVIDER: 'calico',
+        [UNMANAGED_CLUSTER_FIELDS.CONTROL_PLANE_NODE_COUNT]: 1,
+        [UNMANAGED_CLUSTER_FIELDS.WORKER_NODE_COUNT]: 0,
+        [UNMANAGED_CLUSTER_FIELDS.CLUSTER_PROVIDER]: 'kind',
```

Comment:
I added a constants for kubernetes providers strings called K8sProviders.constants.tsx. You can reference 'kind' from that constants file instead of using the string 'kind' here